AUTHORS

```diff
@@ -210,6 +223,7 @@ Denis Defreyne <denis@soundcloud.com>
 Denis Gladkikh <denis@gladkikh.email>
 Denis Ollier <larchunix@users.noreply.github.com>
 Dennis Docter <dennis@d23.nl>
+dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
```

Comment:
Hmm.. should we exclude bots?

Reply:
Yeah, was looking if there's an option for that, but don't think there is; https://git-scm.com/docs/gitmailmap